### DISCUSSION TITLE
How to re-access previously invited git access

### DISCUSSION THREAD

**Discussion by rakshithgit55** [community/community#132233] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have deleted my account from the Git hosting service (like GitHub, GitLab, etc.) and I was previously invited to access a private repository owned by an organization. I want to re-access that repository, but organizations unable to send invitation to me. Is there any option so that I can access there repo.

**Comment by VyvaHart** [comment]
Hi @rakshithgit55

Here are some steps you can try:

- create a new account on the same Git hosting service with the same email address you used previously

- contact the owner/admin of the private repository and inform them of your situation. Provide them with your new account username and email address.

- resend the Invitation: the repository owner/admin will need to resend the invitation to your new account, but.. important moment - ensure they remove any old invitations or references to your previous account

useful tips:

- Update SSH Keys: in case If you use SSH keys for authentication

- Access via Personal Access Token (PAT): If the repository requires a Personal Access Token (PAT) for access, make sure to generate a new token from your account settings and use it for authentication.

good luck!
